### ISSUE TITLE
linux compilation fails from a path that contains link (capnp limitation)

### ISSUE THREAD

**Issue by DamienLeBars** [alainmarcel/UHDM#374]
the use of [file normalize XXX] (resolving links) in model_gen.tcl generate an error in CAPNP call that use path without links resolving.

"File is not in the current directory and does not match any prefix defined with "
 "--src-prefix. Please pass an appropriate --src-prefix so I can figure out where to "
 "write the output for this file."

**Comment by alaindargelas** [comment]
I'm not seeing this problem, neither the Actions or Travis builds.
Are you sure there is not something else at play here?

**Comment by DamienLeBars** [comment]
yes, on our network, we have user home path that contains link.
daisy is the computer i'm connected on and /scorpion is a link to user shared home disk
```bash
daisy:/scorpion/home/dlb > cd
daisy:/scorpion/home/dlb > pwd
/net/scorpion-home/Scorpion-home/home/dlb
daisy:/scorpion/home/dlb > echo $PWD
/scorpion/home/dlb
```
So capnp see the path like $PWD when [file normalize] return resolved path like pwd.
I understand that it is a quite specific problem!

**Comment by alaindargelas** [comment]
I see.
    On Tuesday, December 1, 2020, 10:31:08 PM GMT-10, DamienLeBars <notifications@github.com> wrote:  
 
 


yes, on our network, we have user home path that contains link.
daisy is the computer i'm connected on and /scorpion is a link to user shared home disk
daisy:/scorpion/home/dlb > cd
daisy:/scorpion/home/dlb > pwd
/net/scorpion-home/Scorpion-home/home/dlb
daisy:/scorpion/home/dlb > echo $PWD
/scorpion/home/dlb
So capnp see the path like $PWD when [file normalize] return resolved path like pwd.
I understand that it is a quite specific problem!

**Comment by DamienLeBars** [comment]
Fixed by [alainmarcel/UHDM#375](https://github.com/alainmarcel/UHDM/pull/375)
